src/components/RichTextEditor/Plugins/ListPlugin/ListItemContentMarkupElement.tsx

```diff
@@ -19,14 +22,25 @@ export const getTextStyle = (styledNode: TNode) => {
 
 export const ListItemContentMarkupElementNode = ({ attributes, children, element }: PlateRenderElementProps) => {
     const { designTokens } = useRichTextEditorContext();
+    const align = element.align as unknown as string;
+    const isEmpty = element.children.every((child) => child.text === '');
 
     return (
         <p
-            style={{ textDecoration: designTokens[getTextStyle(element.children[0])]?.textDecoration }}
-            className={getColumnBreakClasses(element)}
+            className={merge([
+                getColumnBreakClasses(element),
+                align ? justifyClassNames[align] : 'tw-justify-start tw-flex',
+            ])}
             {...attributes}
         >
-            {children}
+            <span
+                className={merge([
+                    isEmpty && 'tw-w-4',
+                    designTokens[getTextStyle(element.children[0])]?.textDecoration === 'underline' && 'tw-underline',
```

Comment:
is it necessary that the underline is applied on the LIC level? how do we deal with other styles?

Reply:
underline was already before handled individually.